Sources/MongoSwift/ChangeStream.swift

```diff
@@ -108,13 +108,11 @@ public class ChangeStream<T: Codable>: CursorProtocol {
         }
     }
 
-    deinit {
```

Comment:
these `deinit`s are now handled by the `deinit` in the internal `Cursor` type so we don't have to call an async method from `deinit`.